### PR TITLE
Added min_gas_price resource blockchain param

### PR DESCRIPTION
**Pull request by liayoo**

### CODE REVIEW

File: blockchain-configs/1-node/blockchain_params.json [hunk 1]
```diff
@@ -80,7 +80,8 @@
     "set_op_list_size_limit": 1500,
     "min_gc_num_siblings_deleted": 100,
     "block_chunk_size": 100000000,
-    "snapshot_chunk_size": 100000000
+    "snapshot_chunk_size": 100000000,
```
Comment (platfowner) [inline blockchain-configs/1-node/blockchain_params.json:83]: ah, 100mb could be too big for snapshot sharing between blockchain peers.
if this value is small enough (e.g. 1mb), the saved snapshot file can be directly streamed between peers **without modification**.
what do you think?

Comment (liayoo) [reply blockchain-configs/1-node/blockchain_params.json:83]: 1MB sounds good to me~

File: db/index.js [hunk 2]
```diff
@@ -1700,6 +1700,14 @@ class DB {
           TxResultCode.TX_NO_TX_BODY,
           `[${LOG_HEADER}] Missing tx_body: ${JSON.stringify(tx)}`, 0);
     }
+    const gasPrice = tx.tx_body.gas_price;
+    const minGasPrice = DB.getBlockchainParam('resource/min_gas_price', blockNumber, this.stateRoot);
+    if (!CommonUtil.isInteger(gasPrice) || (gasPrice !== 0 && gasPrice < minGasPrice)) {
```
Comment (platfowner) [inline db/index.js:1705]: what about checking `gasPrice < 0` case as well?

File: blockchain-configs/1-node/blockchain_params.json (hunk 1 above)
Comment (liayoo) [reply blockchain-configs/1-node/blockchain_params.json:83]: I'll reduce the block chunk size to 1mb as well then~ (although it's not being used right now..)

File: db/index.js (hunk 2 above)
Comment (liayoo) [reply db/index.js:1705]: gasPrice < minGasPrice should cover it unless we set the min_gas_price to a negative number? should we still add the check to be extra careful?

File: blockchain-configs/1-node/blockchain_params.json [hunk 3]
```diff
@@ -80,7 +80,8 @@
     "set_op_list_size_limit": 1500,
     "min_gc_num_siblings_deleted": 100,
     "block_chunk_size": 100000000,
```
Comment (platfowner) [inline blockchain-configs/1-node/blockchain_params.json:82]: we don't need this any more? 
can this value just deleted for now?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!!

File: blockchain-configs/1-node/blockchain_params.json (hunk 3 above)
Comment (liayoo) [reply blockchain-configs/1-node/blockchain_params.json]: done~

**Pull request merged**